journal-letters/modernized/03-chapter-1.adoc

```diff
@@ -452,8 +452,8 @@ destitute and without a habitation.
 And then we saw our sacrificing and our sinning to be alike in the sight of God;
 for our prayers were rejected, and all loathed,
 because both were done in one nature and from one and the same seed and corrupt heart;
-and, therefore, it came to be with us as with Judah of old, as may be read Isa.
-1 and Isa. 46:3. where the Lord told Judah, their killing an ox,
+and, therefore, it came to be with us as with Judah of old, as may be read in
+Isa.1 and Isa. 66:3, where the Lord told Judah, their killing an ox,
```

Comment:
This is right, but I think we just need a space between Isa. and 1 (like there is a space between Isa. and 63)